### ISSUE TITLE
Working hardwares

### ISSUE THREAD

**Issue by talpar** [RyzenPan/Dell-3576-hackintosh-EFI#2]
what hardware is working?

**Comment by wuyscc** [comment]
Hmmm, I tested with my laptop. Touchpad and headphone jack don't work because it isn't patched.

**Comment by RyzenPan** [comment]
Touchpad can run，but it is hard-to-use ,so I close it.I not to test headphone ,I think as the system is stable, that's enough

**Comment by wuyscc** [comment]
> Touchpad can run，but it is hard-to-use ,so I close it.I not to test headphone ,I think as the system is stable, that's enough.

It stable I think, btw I hope you update to newest clover version. The version of yours is old. THank you very much.

**Comment by nv-disable** [comment]
![image](https://user-images.githubusercontent.com/64586423/82366908-a3bba680-9a1b-11ea-9290-506500c13006.png)
I had managed to make it work .

**Comment by wuyscc** [comment]
> ![image](https://user-images.githubusercontent.com/64586423/82366908-a3bba680-9a1b-11ea-9290-506500c13006.png)
> I had managed to make it work .

Hope you share EFI soon. Btw, congratulation

**Comment by nv-disable** [comment]
> > ![image](https://user-images.githubusercontent.com/64586423/82366908-a3bba680-9a1b-11ea-9290-506500c13006.png)
> > I had managed to make it work .
> 
> Hope you share EFI soon. Btw, congratulation

Hello. Sorry for late reply. I'll send my EFI soon. :)

**Comment by nv-disable** [comment]
~~[I2C DELL083F TPD0(I2C0).zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4673238/I2C.DELL083F.TPD0.I2C0.zip)
Don't use patched DSDT. Use only my SSDT's + kexts + config patches.
I'll share my whole EFI soon because I need to fix something...~~

Here is my EFI. 
[CLOVER (FIX).zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4673343/CLOVER.FIX.zip)
```diff
#(UPD: I updated the drivers on the touchpad to the latest version. Now it works even better)
```

Also I recommend to disable cfg lock and increase DVMT via Modded grub (I can write a guide).
![image](https://user-images.githubusercontent.com/64586423/82752456-d2909e80-9dc6-11ea-9428-4c712c711fdd.png)

@RyzenPan @JustHmmmm

**Comment by wuyscc** [comment]
> ~[I2C DELL083F TPD0(I2C0).zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4673238/I2C.DELL083F.TPD0.I2C0.zip) Don't use patched DSDT. Use only my SSDT's + kexts + config patches. I'll share my whole EFI soon because I need to fix something...~
> 
> Here is my EFI.
> [CLOVER (FIX).zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4673343/CLOVER.FIX.zip)
> 
> ```diff
> #(UPD: I updated the drivers on the touchpad to the latest version. Now it works even better)
> ```
> 
> Also I recommend to disable cfg lock and increase DVMT via Modded grub (I can write a guide).
> ![image](https://user-images.githubusercontent.com/64586423/82752456-d2909e80-9dc6-11ea-9428-4c712c711fdd.png)
> 
> @RyzenPan @JustHmmmm

Thank you very much. I hope there will be a guide soon.

**Comment by nv-disable** [comment]
Almost everything is works in my CLOVER (incl. 3.5mm audio, bluetooth, camera and other), but I'll migrate to to Skylake framebuffer. SKL framebuffer doesn't have artefacts, that KBL have (e.g. dotty screen. And also no more crashes when connecting HDMI (10.15.5) when using KBL framebuffer.)

Why not Coffee Lake framebuffer? Because it's have tearing on laptops' display when TV is connected via HDMI (HDMI hotplug results restarting Graphics server too, thats annoying) and also no brightness after sleep.

So, wait for updates :). Skylake is out better choice!

**Comment by nv-disable** [comment]
So. Here is my new EFI with SkyLake FrameBuffer.  [CLOVER.zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4705703/CLOVER.zip)
![image](https://user-images.githubusercontent.com/64586423/83329846-62858b00-a294-11ea-8daf-277ce2bec09f.png)

So, try this someone and report me if everything is ok.

**Comment by talpar** [comment]
i have a question. is wifi working?

**Comment by nv-disable** [comment]
> i have a question. is wifi working?

No. QCA9377 is unsupported. Only Bluetooth is working. But Idk maybe you have another module.

**Comment by nv-disable** [comment]
[EFI.zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4846837/EFI.zip)
New OpenCore EFI for Dell 3576. Now I'm using KabyLake Graphics driver, but another framebuffer.
![image](https://user-images.githubusercontent.com/64586423/86032716-73b2ea80-ba40-11ea-9582-7f5dc8d5642d.png)

**Comment by nv-disable** [comment]
![image](https://user-images.githubusercontent.com/64586423/86627489-16cfab00-bfd1-11ea-9ca2-1768e5b944de.png)
I'll just leave it here.

**Comment by wuyscc** [comment]
That's reallly cool. I have a question. Which wifi adapter you are using?

**Comment by nv-disable** [comment]
> Which wifi adapter you are using?

I'm not using Wi-Fi in macOS. My laptop is always connected to Internet via Ethernet cable.

**Comment by wuyscc** [comment]
> I'm not using Wi-Fi in macOS. My laptop is always connected to Internet via Ethernet cable.
Oh, btw thanks for share EFI :1st_place_medal:

**Comment by wuyscc** [comment]
> [EFI.zip](https://github.com/RyzenPan/Dell-3576-hackintosh-EFI/files/4846837/EFI.zip)
> New OpenCore EFI for Dell 3576. Now I'm using KabyLake Graphics driver, but another framebuffer.
> ![image](https://user-images.githubusercontent.com/64586423/86032716-73b2ea80-ba40-11ea-9582-7f5dc8d5642d.png)

With me CLOVER is working but when use OpenCore, I stuck at boot screen :(

**Comment by lookkaz** [comment]
I have this exactly laptop (inspiron 3576) and never done any hackintosh before, so I have some questions:

1) Which version of macOS should be most stable? Will Catalina work well? 

2) Is the hardware working with this EFI? 

3) Do you have any guide for this laptop?

ps: sorry about my english, it is very bad I know.
